File: 백준/Bronze/1977. 완전제곱수/완전제곱수.js
const [M, N] = require('fs').readFileSync(0).toString().trim().split('\n').map(Number);
const result = [];

for (let num = M; num <= N; num++) {
    if (Math.floor(Math.sqrt(num)) ** 2 === num) {
        result.push(num);
    }
}

if (result.length === 0){
    console.log(-1);
} else {
    console.log(result.reduce((acc, cur) => acc + cur, 0));
    console.log(Math.min(...result));
}

File: 백준/Bronze/1977. 완전제곱수/완전제곱수.py
M = int(input())
N = int(input())
result = []

for num in range(M, N + 1):
    if int(num ** 0.5) ** 2 == num:
        result.append(num)

if len(result) == 0:
    print(-1)
else:
    print(sum(result))
    print(min(result))